src/org/chartjs/PieDAOChartView.js

```diff
@@ -52,6 +50,9 @@ foam.CLASS({
             config.data = {
               datasets: [{
                 data: Object.keys(sink.groups).map(key => sink.groups[key].value),
+                backgroundColor: [
```

Comment:
This will make all the content the same colour and I'm not sure we want that. We should make this configurable in some way.

Reply:
yeah I'll remove this here I handle it in my multiple currencies PR